src/pages/components/tile/usage.mdx

```diff
@@ -498,6 +505,25 @@ what content it contains but all types of tiles can take focus.
 | Return or Enter | Return or enter will open the tile (if the tile itself is clickable). If the tile is not clickable but has interactive elements, their corresponding actions are performed. |
 | Space           | If the tile is selectable, the space bar will toggle tile selection.                                                                                                        |
 
+## AI presence
+
+Tile has a modification that takes on the AI visual styling when the AI slug is
+present in the container. The AI variants function the same as the normal
+versions except with the addition of the AI slug which is both a visual
+indicator and the trigger for the explainability popover.
+
+More information about designing for AI guidelines is coming soon.
+
+<Row>
+<Column colLg={8}>
+
+![Tile with AI presence](images/tile-AI-presence-usage.png)
+
+</Column>
+</Row>
+
+<Caption>Tile with AI presence</Caption>
```

Comment:
I don't think you need this caption but if you're going to have it I would be more specific with it so it more accurately describes the image.
```suggestion
<Caption>Base, clickable, single-select and multi-select tiles with AI presence</Caption>
```

Reply:
Oh I thought it was a single image, so thought of adding the caption. 
Will remove it in that case